tiledb/cloud/file.py

```diff
@@ -53,6 +91,19 @@ def export_file(
 
         api_instance = client.client.file_api
 
+        local = True
+        if (
+            uri.startswith("tiledb://")
```

Comment:
Would it be better to check if there is no `://` at all or it starts with `file://`?

```python
local = uri.startswith('file://`) or '://' not in uri
```

Reply:
@thetorpedodog we really need to expose a VFS tiledb setting for this since we have this inside the core code. For now I was just looking for a quick fix. I'd be fine with your suggested adjustment! Do you mind pushing the change to this branch?

Reply:
Done and also rebased to fix the conflict.